Formula/minetest.rb

```diff
@@ -7,9 +7,19 @@ class Minetest < Formula
     url "https://github.com/minetest/minetest/archive/5.5.1.tar.gz"
     sha256 "dc0ae5188ef351db85c38b27f38f8549b133ed82aa46daea6deee148aa3454f4"
 
+    patch do
+      url "https://github.com/minetest/minetest/commit/951604e29ff9d4b796003264574e06031c014a3f.patch?full_index=1"
+      sha256 "46ca51997cadba9ec714de1988097ae37dec013def1cc4ae560d8de6b0f1d0bc"
+    end
```

Comment:
Can we add a comment here explaining what this patch is for and when it can be removed?

Reply:
added comment